src/integrators/inc_hyperelastic_integrator.hpp

```diff
@@ -33,7 +33,7 @@ class IncrementalHyperelasticIntegrator : public mfem::NonlinearFormIntegrator {
 
  public:
   /** @param[in] m  HyperelasticModel that will be integrated. */
-  IncrementalHyperelasticIntegrator(mfem::HyperelasticModel* m) : model_(m) {}
+  explicit IncrementalHyperelasticIntegrator(mfem::HyperelasticModel* m) : model_(m) {}
```

Comment:
Usually you want single-argument constructors to be `explicit`.  The [isocpp guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-explicit) have a section on it, but the main idea is that you want to avoid implicit conversions, e.g., if you have a function that takes an `mfem::Vector` (which has an integer size ctor), you could accidentally pass in the scalar value "4" and not catch the error until runtime since it would get promoted to a vector of size 4.